tests/auth_rust/tests/test_auth_demo.rs

```diff
@@ -0,0 +1,162 @@
+#![allow(unused_imports)]
+#![allow(dead_code)]
+
+use ckb_crypto::secp::{Generator, Pubkey};
+use ckb_error::prelude::thiserror::private::AsDynError;
+use ckb_script::TransactionScriptsVerifier;
+use log::{Level, LevelFilter, Metadata, Record};
+use misc::{
+    assert_script_error, build_resolved_tx, debug_printer, gen_args, gen_consensus, gen_tx,
+    gen_tx_env, gen_tx_with_grouped_args, sign_tx, AlgorithmType, AuthErrorCodeType,
+    DummyDataLoader, EntryCategoryType, TestConfig, MAX_CYCLES,
+};
+use rand::{thread_rng, Rng};
+
+mod misc;
+
+#[test]
+fn ckb_verify() {
+    let mut data_loader = DummyDataLoader::new();
+    let privkey = Generator::random_privkey();
+
+    let config = TestConfig::new(
+        AlgorithmType::Ckb,
+        EntryCategoryType::DynamicLinking,
```

Comment:
The "exec" method should be tested too: more cases.

I think we need to test "exec" mainly. Then append some "dynamic linking" test cases which is not important as "exec".